src/test/java/BogdanSandu/tests/Tema6/HerokuForms.java

```diff
@@ -0,0 +1,83 @@
+package BogdanSandu.tests.Tema6;
+
+import BogdanSandu.driver.BrowserManager;
+import BogdanSandu.utils.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.print.PrintOptions;
+import org.openqa.selenium.support.ui.Select;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import static BogdanSandu.tests.Tema6.HerokuAlerts.tryToFindElement;
+
+public class HerokuForms {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) {
+        navigateToHerokuFormsPage();
+        try {
+            fillFormDetails();
```

Comment:
Do you have any exceptions that can be thrown inside this try? That means to identify in one of your methods an element with a wrong id/selector/xpath. ;)